### PR TITLE
Add cleanups for stale IP addresses on test nodes

### PR DESCRIPTION
**Pull request by chrisvanheuveln**
Beaker failures due to conflicts with leftover 192.168.x.x interfaces on dirty testbeds.

Rubocop clean, affected beaker tests clean.

### CODE REVIEW

File: tests/beaker_tests/lib/utilitylib.rb [hunk 1]
```diff
@@ -308,6 +308,24 @@ def node_feature_cleanup(agent, feature, stepinfo='feature cleanup',
...
+  # configuration (mgmt will not appear).
+  on(agent, show_cmd)
+  clean = stdout.split("\n").map do |line|
+    "interface #{Regexp.last_match[:intf]} ; no ip addr" if
+      line[/^(?<intf>\S+)\s+\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/]
+  end.compact
+  return if clean.empty?
+  clean = clean.join(' ; ').prepend('conf t ; ')
+  logger.debug("#{stepinfo} Clean string:\n#{clean}")
+  on(agent, UtilityLib.get_vshell_cmd(clean), acceptable_exit_codes: [0, 2])
```
Comment (mikewiebe) [inline tests/beaker_tests/lib/utilitylib.rb]: Add a note to clarify why 0 and 2 are valid exit codes.

File: tests/beaker_tests/loopbackintf/loopbackintf_provider_defaults.rb [hunk 2]
```diff
@@ -66,6 +66,9 @@
 test_name "TestCase :: #{testheader}" do
   # @step [Step] Sets up switch for provider test.
   step 'TestStep :: Setup switch for provider test' do
+    # Remove any stale interface IP addresses from testbed.
+    interface_ip_cleanup(agent)
```
Comment (mikewiebe) [inline tests/beaker_tests/loopbackintf/loopbackintf_provider_defaults.rb]: You don't need to call this as `Utilitylib.interface_ip_cleanup` ?

File: tests/beaker_tests/lib/.#utilitylib.rb [hunk 3]
```diff
@@ -0,0 +1 @@
+cvanheuv@rtp-ads-504.6641:1430959141
```
Comment (robert-w-gries) [inline tests/beaker_tests/lib/.#utilitylib.rb]: Was this supposed to be committed?

File: tests/beaker_tests/loopbackintf/loopbackintf_provider_defaults.rb (hunk 2 above)
Comment (chrisvanheuveln) [inline tests/beaker_tests/loopbackintf/loopbackintf_provider_defaults.rb]: It's not in the module.

File: tests/beaker_tests/lib/utilitylib.rb (hunk 1 above)
Comment (chrisvanheuveln) [inline tests/beaker_tests/lib/utilitylib.rb]: done

File: tests/beaker_tests/lib/.#utilitylib.rb (hunk 3 above)
Comment (glennmatthews) [inline tests/beaker_tests/lib/.#utilitylib.rb]: We may want to add `.#*` to `.gitignore` to avoid this in the future.

**Pull request merged**